BosTau.py

```diff
@@ -0,0 +1,211 @@
+import logging
+
+import msprime
+
+import stdpopsim
+
+logger = logging.getLogger(__name__)
+
+###########################################################
+#
+# Genome definition
+#
+###########################################################
+
+
+# Create a string of chromosome lengths for easy parsing
+_chromosome_data = """\
+chr1    158534110
+chr2    136231102
+chr3    121005158
+chr4    120000601
+chr5    120089316
+chr6    117806340
+chr7    110682743
+chr8    113319770
+chr9    105454467
+chr10   103308737
+chr11   106982474
+chr12   87216183
+chr13   83472345
+chr14   82403003
+chr15   85007780
+chr16   81013979
+chr17   73167244
+chr18   65820629
+chr19   63449741
+chr20   71974595
+chr21   69862954
+chr22   60773035
+chr23   52498615
+chr24   62317253
+chr25   42350435
+chr26   51992305
+chr27   45612108
+chr28   45940150
+chr29   51098607
+chrX    139009144
+chrMT   16338
+"""
+# Parse list of chromosomes into a list of Chromosome objects which contain the
+# chromosome name, length, mutation rate, and recombination rate
+_chromosomes = []
+
+for line in _chromosome_data.splitlines():
+    name, length = line.split()[:2]
+    _chromosomes.append(stdpopsim.Chromosome(
+        id=name, length=int(length),
+        mutation_rate=1e-8,
+        recombination_rate=1e-8))
+
+# A citation for the chromosome parameters. Additional citations may be needed if
+# the mutation or recombination rates come from other sources. In that case create
+# additional citations with the appropriate reasons specified (see API documentation
+# for stdpopsim.citations)
+
+_assembly_citation = stdpopsim.Citation(
+    doi="FILL ME",
+    year="FILL ME",
+    author="Author et al.",
+    reasons={stdpopsim.CiteReason.ASSEMBLY})
+
+# Create a genome object
+
+_genome = stdpopsim.Genome(
+    chromosomes=_chromosomes,
+    assembly_citations=[_assembly_citation])
+
+# Create a Species Object
+_gen_time_citation = stdpopsim.Citation(
+    doi="",
+    year="",
+    author="",
+    reasons={stdpopsim.CiteReason.GEN_TIME})
+
+_pop_size_citation = stdpopsim.Citation(
+    doi="",
+    year="",
+    author="",
+    reasons={stdpopsim.CiteReason.POP_SIZE})
+
+_species = stdpopsim.Species(
+    id="BosTau",
+    name="Bos Taurus",
+    common_name="B. taurus",
+    genome=_genome,
+    generation_time=5,
+    generation_time_citations=[_gen_time_citation],
+    population_size=62000,
+    population_size_citations=[_pop_size_citation]
+)
+
+stdpopsim.register_species(_species)
+
+
+###########################################################
+#
+# Demographic models
+#
+###########################################################
+
+def _iid():
+    id = "IonaInferredDemography"
+    description = "Iona MacLeod's Inferred Demographic Model for Bos Taurus"
+    long_description = """
+    The Runs of Homozygosity-based infer of Demography from MacLeod et al. 2013.
+    """
+    populations = [
+        stdpopsim.Population(id="FILL ME", description="FILL ME"),
```

Comment:
Please add population id and description. See other implemented models for examples.